services/web/server/src/simcore_service_webserver/resource_usage/api.py

```diff
@@ -36,6 +36,7 @@ async def add_credits_to_wallet(
     payment_id: PaymentID,
     created_at: datetime,
 ) -> None:
+    assert osparc_credits != 0  # nosec
```

Comment:
can this actually happen?

Reply:
osparc credits can never be zero, that is why i assert it in here. defensive programming :-)